src/components/data/DataTable.tsx

```diff
@@ -288,9 +288,7 @@ export default function DataTable<Row>({
         onKeyDown={event => handleKeyDown(event, row)}
       >
         {fields.map(field => (
-          <TableCell key={field}>
-            {renderItem(row, field as keyof Row)}
-          </TableCell>
+          <TableCell key={field}>{renderItem(row, field)}</TableCell>
```

Comment:
Not related with this PR in particular, but I noticed this casting to `keyof Row` is no longer needed, as `field` is already typed as `keyof Row`.